.dockerignore

```diff
@@ -0,0 +1,10 @@
+.git*
+bin
+CODE_OF_CONDUCT.md
+CONTRIBUTING.md
+Dockerfile
+.dockerignore
+LICENSE
+README.md
+coverage.txt
+.github
```

Comment:
Forgot to mention. Add .travis.yml and .golang-ci.yml to this list as well. Just noticed that it gets copied.